src/status/status.py

```
@@ -231,26 +310,45 @@ def _get_internal_deployment_status(deployment_name: str, analysis_id: str) -> s
+        client = Client(base_url=f"http://nginx-{deployment_name}:{PORTS['nginx'][0]}")
         try:
-            response = Client(base_url=f"http://nginx-{deployment_name}:{PORTS['nginx'][0]}").post("/analysis/token_refresh",
-                                                                                                   json={'token': keycloak_token},
-                                                                                                   headers=[('Connection', 'close')])
+            response = client.post("/analysis/token_refresh",
+                                   json={'token': keycloak_token},
+                                   headers=[('Connection', 'close')])
             response.raise_for_status()
         except HTTPStatusError as e:
-            print(f"Error: Failed to refresh keycloak token in deployment {deployment_name}.\n{e}")
+            logger.error(f"Failed to refresh keycloak token in deployment {deployment_name}: {repr(e)}")
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)